client/components/upgrades/google-apps/google-apps-dialog/product-details.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+
+class GoogleAppsProductDetails extends Component {
+	static propTypes = {
+		annualPrice: PropTypes.string.isRequired,
+		monthlyPrice: PropTypes.string.isRequired
+	};
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<div className="google-apps-dialog__product-details">
+				<div className="google-apps-dialog__product-intro">
+					<h3 className="google-apps-dialog__product-name">
+						{ /* Intentionally not translable as it is a brand name and Google keeps it in English */ }
+						<span className="google-apps-dialog__product-logo">G Suite</span>
+					</h3>
+
+					<p>
+						{
+							translate(
+								"We've partnered with Google to offer you email, storage, docs, calendars " +
+								'and more, integrated with your site.'
+							)
+						}
+					</p>
+
+					<h4 className="google-apps-dialog__price-per-user">
+						{ translate( '%(monthlyPrice)s per user / month', { args: { monthlyPrice: this.props.monthlyPrice } } ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( '%(price)s per user / month ' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).